### ISSUE TITLE
Debugging in iOS?

### ISSUE THREAD

**Issue by marcmolla** [pybee/toga#76]
This question is not only related with Toga, but it comes to my mind while I was working on #75, so I put the question here.

I have the feeling that if I were able to debug my app (breakpoints, step by step execution, variable watch, etc.), I would be easy to find the bug. I am not expert on this, but at the end the iOS is executing python and I guess that somehow we can run it in debug mode and we can connect our IDE to the debugger...

I just want to check if this is possible or I am saying something ridiculous :)

**Comment by freakboy3742** [comment]
Yes - this is absolutely possible. Since we're using the normal Python runtime, you can inject debugging elements into the runtime the same as you would on the desktop. 

Ultimately, I want to be able to connect Bugjar to a running iOS (or any other) app.

**Comment by marcmolla** [comment]
I looked into the pydev remote debugging server and yes, it is possible :). I sent a PR (#82) with a tutorial about how to configure it in an iOS environment.
